rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/mute/muterunner.kt

```diff
@@ -84,11 +86,13 @@ fun showSourceAbsolutePath(): Boolean = "true" == System.getProperty("showSource
 fun executeWithMutes(
     path: Path,
     verbose: Boolean = false,
-    logConfigurationFile: File?
+    logConfigurationFile: File?,
+    programFinders: List<RpgProgramFinder> = emptyList(),
+    output: PrintStream? = null
 ): ExecutionResult {
     var failed = 0
     var executed = 0
-    var resolved: List<MuteAnnotationResolved> = listOf()
+    var resolved: List<MuteAnnotationResolved>
```

Comment:
why removing the default value?

Reply:
It becomes useless (so it generates a warning) based  on the next change, i.e. result?.root?.muteContexts?.size ?: 0